packages/office-ui-fabric-react/src/components/Button/examples/fluent/FluentTheme.ts

```diff
@@ -26,16 +26,27 @@ export const FluentButtonTheme = {
 
     // },
     tiny: {
-      root: { fontSize: '20%' }
+      true: {
+        root: { fontSize: '20%' }
+      }
     },
     large: {
-      root: { fontSize: '400%' }
+      true: {
+        root: { fontSize: '400%' }
+      }
+    },
+    size: {
```

Comment:
Alternative proposal would be using something like:
```
'size.m': {
  //some styles...
},
'size.l': {
  // some styles...
}
```
Or some other similar format, because I think most of them are boolean, but I don't have strong opinion on this, just an option.